Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,24 +1,34 @@
 import React from 'react';
 import './App.css';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
+import Footer from './components/FooterComponents/Footer';
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className='container'>
+      <HeaderContainer imageLink={thumbnailImagesLinks[0]} title={headerTitleTitles[0]} text={headerTexts[0]} />
```

Comment:
For something like this I would recommend taking some time to look at the type of data you want to present. In this case, the data would have looked much cleaner as an object. Then you could just reference with something like data.imageLinks. You could also just put them in template literal strings so that you don't have to worry about bracket notation. This would make your code more readable.